go/test/endtoend/vtgate/schematracker/sharded_prs/st_sharded_test.go

```diff
@@ -156,7 +172,7 @@ func TestMain(m *testing.M) {
 			return 1
 		}
 
-		err = waitForVTGateAndVTTablet()
+		err = clusterInstance.WaitForVTGateAndVTTablets(5 * time.Minute)
```

Comment:
this looks like too much time to wait to fail the CI run eventually.

Reply:
I didn't come up with that time, I merely migrated the code to a utility function. That time came from here: https://github.com/vitessio/vitess/blame/c89fa57b6330a4a878aea82350f8525b50ad4b77/go/test/endtoend/vtgate/schematracker/sharded_prs/st_sharded_test.go#L196